### PR TITLE
Distinguish realized and unrealized P/L in the taker UI

### PR DESCRIPTION
**Pull request by da-kami**
image

### CODE REVIEW

File: taker-frontend/src/components/History.tsx [hunk 1]
```diff
@@ -93,13 +93,18 @@ const CfdDetails = ({ cfd, connectedToMaker, displayCloseButton }: CfdDetailsPro
...
     let profitColors = !cfd || !cfd.profit_btc || cfd.profit_btc === 0
-            || [StateKey.REJECTED, StateKey.SETUP_FAILED].includes(cfd.state.key)
+            || failedCfd
         ? { light: "gray.500", dark: "gray.400" }
         : cfd.profit_btc > 0
         ? { light: "green.600", dark: "green.300" }
         : { light: "red.600", dark: "red.300" };
 
+    let profitLabel = isClosed(cfd) ?
+        failedCfd ? "Missed P/L 😭" : "Realized P/L" : "Unrealized P/L";
```
Comment (thomaseizinger) [inline taker-frontend/src/components/History.tsx]: 😂😂😂

Definitely not enough emojis in this codebase.

Comment (thomaseizinger) [reply taker-frontend/src/components/History.tsx]: /me looking for a usecase for 💩

Comment (thomaseizinger) [reply taker-frontend/src/components/History.tsx]: Note: Making these things components with early return might be easier to understand.

**Review by thomaseizinger — APPROVED** [review]

**Comment by thomaseizinger** [comment]
Should we also just say "Unrealized profit" and "Unrealized loss" instead of P/L? It would be very easy to determine ...

**Review by bonomat — APPROVED** [review]
👍

**Review by klochowicz — APPROVED** [review]

File: taker-frontend/src/components/History.tsx [hunk 2]
```diff
@@ -93,13 +93,19 @@ const CfdDetails = ({ cfd, connectedToMaker, displayCloseButton }: CfdDetailsPro
             isForceCloseButton={true}
         />;
 
+    let failedCfd = [StateKey.REJECTED, StateKey.SETUP_FAILED].includes(cfd.state.key);
```
Comment (thomaseizinger) [inline taker-frontend/src/components/History.tsx:96]: We can probably learn a lot from this as to what the `CfdState` should actually be. Likely, we are exposing too many details here.

**Pull request merged**